its_on/models.py

```diff
@@ -17,6 +18,13 @@
     sa.Column('groups', postgresql.ARRAY(sa.String(255))),
     sa.Column('version', sa.Integer, nullable=True),
     sa.Column('comment', sa.Text),
+    sa.Column('created_at', sa.DateTime, default=lambda: datetime.datetime.now(), nullable=True),
```

Comment:
You'd better use `server_default` and `functions.now()` to make sure datetimes are consistent and not depend on application server's time settings. See [here](https://stackoverflow.com/a/33532154/4113568) for more details

Reply:
`default=datetime.datetime.utcnow`

Reply:
> `default=datetime.datetime.utcnow`

lambda for datetime mocking


> You'd better use `server_default` and `functions.now()` to make sure datetimes are consistent and not depend on application server's time settings. See [here](https://stackoverflow.com/a/33532154/4113568) for more details

1. server_onupdate don't set "ON UPDATE CURRENT_TIMESTAMP" - https://docs.sqlalchemy.org/en/13/core/defaults.html#marking-implicitly-generated-values-timestamps-and-triggered-columns (it working only for mysql)
2. Using datetime i control default values with code and can mock it

Reply:
> server_onupdate don't set "ON UPDATE CURRENT_TIMESTAMP"

I think you've misread the docs - they say the opposite: this basically doesn't work for MySQL out of the box (although there's a workaround for it) but works for everything else. As for the second argument - you technically can use freezegun to control time during tests so mocking it is not really needed.

That said I think this is generally better to use db time but if you think that server time works here as well - i'm fine with that?